src/sentry/static/sentry/app/views/performance/transactionSummary/userStats.tsx

```diff
@@ -84,6 +84,9 @@ function UserStats({totals, location, organization, transactionName}: Props) {
       <div>
         <SectionHeading>{t('Apdex Score')}</SectionHeading>
         <StatNumber>{apdex}</StatNumber>
+        <Link to={`/settings/${organization.slug}/performance/`}>
+          <SectionValue>Max. {threshold}ms</SectionValue>
```

Comment:
The threshold isn't a max though. Could we use 'Threshold' here instead?

Reply:
Sure - will change